src/Ravello.js

```diff
@@ -3,16 +3,14 @@
 const methods = require('./methods');
 const request = require('./request');
 const waiters = require('./waiters');
-const configure = require('./conf').configure;
+const constants = require('./constants');
 
-// Compose library methods
 const composeMethod = ({ method, path }) => (body) => request({ path, method, body });
 
 const builtMethods = {};
 
 Object.keys(methods).forEach((key) => { builtMethods[key] = composeMethod(methods[key]); });
 
-// Compose export
-const Ravello = Object.assign({ configure }, builtMethods, waiters);
+const Ravello = Object.assign({}, builtMethods, waiters, constants);
```

Comment:
Looks like this will break the promise config since we're no longer exporting the `configure` method.